src/wp-includes/Text/Diff.php

```diff
@@ -276,7 +276,7 @@ function _check($from_lines, $to_lines)
 
         $prevtype = null;
         foreach ($this->_edits as $edit) {
-            if ($edit instanceof $prevtype) {
+            if ($prevtype !== null && $edit instanceof $prevtype) {
```

Comment:
Yes, this is a needed defensive change to protect against throwing a fatal error.

Since PHP 5, if `$prevtype` is `null`, the following fatal error is thrown:

{{{
Fatal error: Uncaught Error: Class name must be a valid object or a string
}}}

https://3v4l.org/rQ0s4#veol

Reply:
Created https://core.trac.wordpress.org/ticket/62083 for this specific bugfix.

Reply:
Committed via https://core.trac.wordpress.org/changeset/59070 ✅